spec/f-utils/f-utils-spec.js

```diff
@@ -0,0 +1,140 @@
+'use babel'
+
+// eslint-disable-next-line no-unused-vars
+import 'jasmine-fix'
+import {
+  fromMapToObj,
+  fromObjToMap,
+  isNotNil,
+  deepProp,
+  keyedFilter,
+  keyedMap,
+} from '../../src/f-utils'
+
+
+describe('fromMapToObj', () => {
+  it('transforms a Map to an Object dictionary', () => {
+    const map = new Map([['a', 'some string'], ['b', 10], ['c', {}]])
```

Comment:
Values: `null`, `undefined`, `[]`?

Keys: All the things that Map's can use as keys but Arrays can't. (Mostly kidding, but what _should_ it do here?)

Reply:
Hmm. Didn't consider that because I can't think of any reason I would want to use `null`, `undefined` or `[]` as a key.

Reply:
What do _you_ think it should do here?

Reply:
Oh, I meant those as values, and if you have no use for them (and can't use them anyway) I would `throw`.